terraform/aws/.envrc

```diff
@@ -1,6 +1,6 @@
 # Terraform
-export TF_WORKSPACE=development
-export TF_VAR_default_role=operator
+export TF_WORKSPACE=mlpab
+export TF_VAR_default_role=breakglass
```

Comment:
I don't think it particularly matters, but should this still be development/operator?